src/Riok.Mapperly/Descriptors/MappingBuilders/EnumMappingBuilder.cs

```diff
@@ -78,11 +97,31 @@ private static TypeMapping BuildNameMapping(MappingBuilderContext ctx, bool igno
             .Where(x => x.Target != null)
             .ToDictionary(x => x.Source.Name, x => x.Target!.Name);
 
-        if (enumMemberMappings.Count == 0)
+        if (!enumMemberMappings.Any())
         {
             ctx.ReportDiagnostic(DiagnosticDescriptors.EnumNameMappingNoOverlappingValuesFound, ctx.Source, ctx.Target);
         }
 
+        var missingSourceMembers = sourceFieldsByName.Where(field => !enumMemberMappings.ContainsKey(field.Key));
+        foreach (var member in missingSourceMembers)
+            ctx.ReportDiagnostic(
```

Comment:
This code base uses braces for foreach and ifs, except if it is only a return/throw statement.

Reply:
Fixed.